spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/reactive/DefaultPartHttpMessageReaderCustomizerConfiguration.java

```diff
@@ -0,0 +1,65 @@
+package org.springframework.boot.autoconfigure.web.reactive;
+
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Paths;
+import java.util.function.Consumer;
+
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.boot.context.properties.PropertyMapper;
+import org.springframework.boot.web.codec.CodecCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.annotation.Order;
+import org.springframework.http.codec.multipart.DefaultPartHttpMessageReader;
+import org.springframework.util.unit.DataSize;
+
+@Configuration(proxyBeanMethods = false)
+@EnableConfigurationProperties(MultipartProperties.class)
+class DefaultPartHttpMessageReaderCustomizerConfiguration {
+
+	@Bean
+	@Order(10)
+	CodecCustomizer defaultPartHttpMessageReaderCustomizer(MultipartProperties multipartProperties) {
+		return (configurer) -> configurer.defaultCodecs().configureDefaultCodec((codec) -> {
+			if (!DefaultPartHttpMessageReader.class.isInstance(codec)) {
+				return;
+			}
+			DefaultPartHttpMessageReader defaultPartHttpMessageReader = (DefaultPartHttpMessageReader) codec;
+			boolean streaming = multipartProperties.getStreaming() != null && multipartProperties.getStreaming();
```

Comment:
This is all a bit messy when the underlying target already handles the ignores logic. Very similar concern here as w/ the [default values](https://github.com/spring-projects/spring-boot/pull/26254/files#r621253129).